onnxmltools/utils/float16_converter.py

```diff
@@ -50,20 +52,20 @@ def float16_converter(model_path):
             if hasattr(q, 'graphs'):
                 for n in q.graphs:
                     next_level.append(n)
-            # graph.initializer  (TensorProto)
+            # if q is graph, process graph.initializer(TensorProto), input and output (ValueInfoProto)
```

Comment:
Do we also need to handle intermediate variables (ValueInfoProto's which are neither inputs nor outputs)?